cmake/extensions.cmake

```diff
@@ -1579,16 +1579,6 @@ function(generate_unique_target_name_from_filename filename target_name)
   set(${target_name} gen_${x}_${unique_chars} PARENT_SCOPE)
 endfunction()
 
-# Usage:
-#   add_partition_manager_config(pm.yml)
-#
-# Will add all configurations defined in pm.yml to the global list of partition
-# manager configurations.
-function(add_partition_manager_config config_file)
-  get_filename_component(pm_path ${config_file} REALPATH)
-  set_property(
-    GLOBAL APPEND PROPERTY
-    PM_SUBSYS
-    ${pm_path}
-    )
-endfunction()
+if (ZEPHYR_NRF_MODULE_DIR)
+  include(${ZEPHYR_NRF_MODULE_DIR}/cmake/extensions.cmake)
+endif()
```

Comment:
Is it possible to use `fw-nrfconnect-zephyr` standalone (that is outside NCS scope) ?

If not, and if it should not be so, then we could skip the `if(ZEPHYR_NRF_MODULE_DIR)`

Reply:
See https://github.com/NordicPlayground/fw-nrfconnect-zephyr/pull/279#issuecomment-591828282

Reply:
accepted to keep the test here, for the reason given below.

Reply:
@hakonfam don't know if this PR is still valid, but with the introduction of: https://github.com/nrfconnect/sdk-nrf/commit/c59d4df5c53d14f7bc96a588361ef0f14c989906
then we actually have the possibility of loading the nrf extensions BEFORE Zephyr extensions.cmake.
Guess that would be much nicer :smile: